content/millennium/dstu2/medications/medication-order.md

```diff
@@ -7,6 +7,36 @@ title: MedicationOrder | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Medication Order resource provides orders for all medications along with administration instructions for a patient in both the inpatient and outpatient setting (orders/prescriptions filled by a pharmacy and discharge medication orders). This resource does NOT include historical or home medications documented or reported to be taken by patient, significant other or another provider. 
```

Comment:
convention we've followed so far is we separate the resource name in the overview. And then reference it as the api would elsewhere. Not sure if we're stuck on that convention or not :)